android/app/src/main/java/org/missingmaps/mapswipe/MainActivity.java

```diff
@@ -5,7 +5,6 @@
 import com.facebook.react.ReactActivityDelegate;
 import com.facebook.react.ReactRootView;
 import com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView;
-import org.devio.rn.splashscreen.SplashScreen;
```

Comment:
do we want to have the splashscreen again? Or is there any other reason, why we should remove it?

Reply:
Two different things at play here:
- RN's import system works slightly differently now, so this line is not needed
- the splashscreen package we were using is not supported anymore, so I've removed it (first to check if it was causing the crash, but also to replace it with a properly maintained one, which I'll do asap)